yarn-project/scripts/src/benchmarks/aggregate.ts

```diff
@@ -208,7 +232,11 @@ function processEntry(entry: Stats, results: BenchmarkCollectedResults, fileName
     case 'l2-block-handled':
       return processRollupBlockSynced(entry, results);
     case 'circuit-simulation':
-      return processCircuitSimulation(entry, results);
+      return processCircuitSimulation(entry, results, fileName);
```

Comment:
I'll wait for one run of the benchmark first but I think we'll have to separate all stats coming from the proving benchmark so it doesn't skew the results